pkgs/applications/video/handbrake/default.nix

```diff
@@ -48,6 +48,7 @@ assert stdenv.isDarwin -> AudioToolbox != null
        && Foundation != null
        && libobjc != null
        && VideoToolbox != null;
+assert !stdenv.isDarwin -> numactl != null;
```

Comment:
In my opinion, asserts like this don't prevent any mistake one could conceivably make, but may cause unnecessary headache when doing overrides (see #53487 for details)

Reply:
Is it your opinion that the statement before this one is also superfluous?

Reply:
@bdesham Well, you are the author of 765cd8c2edb8c3406b49f772bb37eaa271c4b2b2, so you might know why it is needed. After a quick check I still don't see a practical use case.

Reply:
This has been rebased and updated to exclude those `assert`s.

Reply:
Thanks! Let's mark this as resolved then. If you come up with a reason why the asserts are useful, I would be curious to know, feel free to ping me.